opal/corelib/basic_object.rb

```diff
@@ -48,21 +48,21 @@ def !=(other)
 
   def instance_eval(*args, &block)
     if block.nil? && `!!Opal.compile`
-      Kernel.raise ArgumentError, "wrong number of arguments (0 for 1..3)" unless (1..3).cover? args.size
+      ::Kernel.raise ::ArgumentError, "wrong number of arguments (0 for 1..3)" unless (1..3).cover? args.size
```

Comment:
Is it required to put `::` everywhere now to avoid calling local constants? (Like potentially generated on the fly `BasicObject::Kernel`)